src/Modifiers/CoreModifiers.php

```diff
@@ -1849,6 +1868,23 @@ public function readTime($value, $params)
         return ceil($words / Arr::get($params, 0, 200));
     }
 
+    /**
+     * Wraps regex matches with <mark> tags.
+     *
+     * @param $value
+     * @param  array  $params
+     * @return string
+     */
+    public function regexMark($value, $params)
+    {
+        $pattern = array_shift($params);
```

Comment:
Probably should bail early in here too if there's no appropriate pattern. Doing `{{ value | regex_mark }}` will mark every character too.